src/main/java/Ui.java

```diff
@@ -0,0 +1,11 @@
+public class Ui {
+    private static String drawBorder(String text) {
+        return "____________________________________________________________\n"
+                + text + "\n"
```

Comment:
I like how Yan Jie has even extracted the UI printing as an object as well! Good job!